lib/features/manual_entry/QA.md

```
@@ -0,0 +1,214 @@
+
+<details><summary>動作確認スクショ</summary>
+
+**確認日: 2026-08-19**
+
+runner の Simulator が英語ロケールのため、英語表示で確認した (manualEntryAmountRequired の en 値 = "Enter an amount of at least 1 yen")。
+
+左: 金額を空のまま "Add transaction" をタップ。金額欄が赤枠になり "Enter an amount of at least 1 yen" が表示され、シートは開いたまま。右: 金額に 0 を入れて再度タップしても同じエラーで登録されない。どちらもシート背後のサマリー Spending は ¥6,480 のまま増えておらず、スナックバーも出ていない。
+
+0 円明細が保存されていないことは、この直後に同じシートを閉じるボタンで閉じた「キャンセル」項目のスクショ (明細 4 件: Salary August / Cash expense / Lawson QA / Past Date Cafe のまま、0 円の行が無い) で確認した。
+
+<img src="https://pub-7f3469dd3e2e445b9b8ec2d1381b5ea8.r2.dev/bannzai/kashakeibo/20260819/653496a2-92ec-4c8d-8344-86ed7256d631.jpg" width="320">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)